xmps/color_selection/line_fits.py

```diff
@@ -0,0 +1,99 @@
+from astropy.io import fits
+import matplotlib
+matplotlib.use('Qt4Agg')
+import numpy as np
+import matplotlib.pyplot as plt
+
+from scipy.optimize import curve_fit
+from scipy import asarray as ar,exp
+import numpy.ma as ma
+
+'''
+This code is for fitting emission lines (gaussian) and measuring their equivalent widths
+'''
+
+Hb=4862.721
```

Comment:
I would much prefer that these values came
from linetools.  Consider that.

Reply:
Agreed. These were simple estimates to feed into ALIS for its fitting procedure.

Reply:
Do are you modifying this?